Makefile

```diff
@@ -50,6 +50,7 @@ CH5xx_ble_firmware_library/StdPeriphDriver/CH58x_usb2hostBase.c \
 CH5xx_ble_firmware_library/StdPeriphDriver/CH58x_spi1.c \
 CH5xx_ble_firmware_library/RVMSIS/core_riscv.c \
 src/main.c \
+leddrv.c
```

Comment:
path issue

Reply:
My bad. But somehow the build still works. 

I'll get it fixed right away.